app/mailers/user_mailer.rb

```diff
@@ -372,8 +372,8 @@ def admin_deleted_work_notification(user, work)
 
   # Sends email to authors when a creation is hidden by an Admin
   def admin_hidden_work_notification(creation_id, user_id)
-    @user = User.find_by_id(user_id)
-    @work = Work.find_by_id(creation_id)
+    @user = User.find_by(id: user_id)
```

Comment:
`find_by_id` was used previously so I kept the same concept (return nil vs raise exception). If `@user` is required later, either would be just fine.